kernelci.org/content/en/_index.md

```diff
@@ -37,6 +37,13 @@ Welcome to the KernelCI documentation website.
   to contribute your results to the KernelCI common database. KCIDB can receive tests results from any CI system.</p>
 </div>
 
+<div class="container border border-primary rounded p-2 mt-3">
+  <h2 class="text-center">
+    <a href="kci-dev">kci-dev</a>
+  </h2>
+  <p class="text-center">Start here if you want to interact with KernelCI instances</p>
```

Comment:
Nit: We should add a full stop like the other bordered entries on this page.

Perhaps we could also make it clearer how the suggested interaction will happen? Maybe "programmatically"?
I'm open to better suggestions though!

```suggestion
  <p class="text-center">Start here if you want to interact with KernelCI instances programmatically.</p>
```